test/e2e/util/cluster.go

```diff
@@ -430,6 +430,41 @@ func APIManagerDeploymentTest(t *testing.T, ns string, retryInterval, timeout ti
 	}
 }
 
+// APIManagerWebhookServiceTest checks the api-manager webhook service deployment.
+func APIManagerWebhookServiceTest(t *testing.T, ns string, retryInterval, timeout time.Duration) {
```

Comment:
@croomes looks like this wasn't added in clusterCSINodeV2_test.go file to be executed.